Tutorial01/tutorial01.rb

```diff
@@ -0,0 +1,12 @@
+for a in 1..6
+
+  print " "*(6-a), "*" * ( ( 2 * a  ) - 1 ) , "\n"
```

Comment:
Use spaces around operators, after commas, colons and semicolon, around { and befor }